infrastructure/template.yaml

```diff
@@ -1330,6 +1330,12 @@ Resources:
             TableName: !Ref NinoUsersTable
         - EventBridgePutEventsPolicy:
             EventBusName: !Ref CheckHmrcEventBus
+        - SSMParameterReadPolicy:
+            ParameterName: check-hmrc-cri-api/contraindicationMappings
+        - SSMParameterReadPolicy:
+            ParameterName: check-hmrc-cri-api/contraIndicatorReasonsMapping
```

Comment:
What's the difference between the `contraindicationMappings` and `contraIndicatorReasonsMapping` SSM params?